esmvaltool/recipes/recipe_ocean_bgc.yml

```diff
@@ -58,20 +58,20 @@ preprocessors:
     custom_order: true
     volume_statistics:
       operator: mean
-    multi_model_statistics:
-      span: overlap
-      statistics: [mean]
-      exclude: ['WOA']
+    #multi_model_statistics:
+    #  span: overlap
+    #  statistics: [mean]
+    #  exclude: ['WOA']

```

Comment:
Multi_model_statistics commented out because this recipe uses one model only I guees. Or is there another reason for that?

Reply:
It is exactly for that ! only one model is used...